core/src/main/java/com/novoda/noplayer/drm/provision/ProvisionExecutor.java

```diff
@@ -25,7 +23,7 @@ public static ProvisionExecutor newInstance() {
         this.capabilities = capabilities;
     }
 
-    public byte[] execute(MediaDrm.ProvisionRequest request) throws IOException, UnableToProvisionException {
+    public byte[] execute(ModularDrmProvisionRequest request) throws IOException, UnableToProvisionException {
         if (isIncapabaleOfProvisioning()) {
```

Comment:
I was sitting here for a good few minutes thinking 🤔  where is the typo, Ah there it is 😄